### PR TITLE
Adding Samples of area chart, 3D area chart, stacked area chart, 100% stacked area chart, 3d 100% stacked area chart, 3d stacked area chart in Charts

### PR DESCRIPTION
**Pull request by sharonaumao**
Added KB documentation samples. The samples are:

- Create-area-chart
- Create-3D-area-chart
- Create-stacked-area-chart
- Create-100-stacked-area-chart
- Create-3D-100-stacked-area-chart
- Create-3D-stacked-area-chart

### CODE REVIEW

File: Charts/Create-area-chart/.NET-Standard/Create-area-chart/Create-area-chart.csproj [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RootNamespace>Create-clustered_bar_chart</RootNamespace>
```
Comment (Ramaraj-Marimuthu) [inline Charts/Create-area-chart/.NET-Standard/Create-area-chart/Create-area-chart.csproj:6]: @sharonaumao - Please use proper namespace here.

File: Charts/Create-stacked-area-chart/.NET-Standard/Create-stacked-area-chart/Create-stacked-area-chart.csproj [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RootNamespace>Create-clustered_bar_chart</RootNamespace>
```
Comment (Ramaraj-Marimuthu) [inline Charts/Create-stacked-area-chart/.NET-Standard/Create-stacked-area-chart/Create-stacked-area-chart.csproj:6]: @sharonaumao - - Please use proper namespace here.

**Review by Ramaraj-Marimuthu — CHANGES_REQUESTED** [review]

File: Charts/Create-area-chart/.NET-Standard/Create-area-chart/Create-area-chart.csproj (hunk 1 above)
Comment (sharonaumao) [reply Charts/Create-area-chart/.NET-Standard/Create-area-chart/Create-area-chart.csproj]: @Ramaraj-Marimuthu -Changed

File: Charts/Create-stacked-area-chart/.NET-Standard/Create-stacked-area-chart/Create-stacked-area-chart.csproj (hunk 2 above)
Comment (sharonaumao) [reply Charts/Create-stacked-area-chart/.NET-Standard/Create-stacked-area-chart/Create-stacked-area-chart.csproj]: @Ramaraj-Marimuthu -Changed

**Review by Ramaraj-Marimuthu — APPROVED** [review]
I don't have any concerns in this code changes.

**Pull request merged**